.github/rawWorkflows/gh-ci-parameterized-flow.txt

```diff
@@ -28,3 +28,18 @@
         uses: gradle/gradle-build-action@v2
         with:
           arguments: "$GradleArguments"
+      - name: Package Build Artifacts
+        if: success() || failure()
+        shell: bash
+        run: |
+          mkdir ${{ github.job }}-artifacts
+          find . -path "**/build/reports/*" -or -path "**/build/test-results/*" > artifacts.list
+          rsync -R --files-from=artifacts.list . ${{ github.job }}-artifacts
+          tar -zcvf ${{ github.job }}-artifacts.tar.gz ${{ github.job }}-artifacts
+      - name: Upload Build Artifacts
+        if: success()
```

Comment:
This is not needed, but not an issue as well

Reply:
Right. It's just to prevent warnings in annotations.